src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/BlockListener.java

```diff
@@ -222,6 +229,53 @@ private void dropItems(BlockBreakEvent e, List<ItemStack> drops) {
         }
     }
 
+    /**
+     * This method checks recursively for any sensitive blocks
+     * that are no longer supported due to this block breaking
+     *
+     * @param block
+     *      The {@link Block} in question
+     * @param count
+     *      The amount of times this has been recursively called
+     */
+    @ParametersAreNonnullByDefault
+    private void checkForSensitiveBlocks(Block block, Integer count) {
+        if (count >= Bukkit.getServer().getMaxChainedNeighborUpdates()) {
+            return;
+        }
+        for (BlockFace face : CARDINAL_BLOCKFACES) {
+            block.setType(Material.AIR, false);
+            if (!isSupported(block.getRelative(face).getBlockData(), block.getRelative(face))) {
+                Block relative = block.getRelative(face);
+                for (ItemStack drop : relative.getDrops()) {
+                    block.getWorld().dropItemNaturally(relative.getLocation(), drop);
+                }
+                checkForSensitiveBlocks(relative, ++count);
+            }
+        }
+    }
+
+    /**
+     * This method checks if the {@link BlockData} would be
+     * supported at the given {@link Block}.
+     *
+     * @param blockData
+     *      The {@link BlockData} to check
+     * @param block
+     *      The {@link Block} the {@link BlockData} would be at
+     * @return
+     *      Whether the {@link BlockData} would be supported at the given {@link Block}
+     */
+    @ParametersAreNonnullByDefault
+    private boolean isSupported(BlockData blockData, Block block) {
+        if (Slimefun.getMinecraftVersion().isAtLeast(MinecraftVersion.MINECRAFT_1_19)) {
+            return blockData.isSupported(block);
+        } else {
+            // TODO: Make 1.16-1.18 version. BlockData::isSupported is 1.19+.
```

Comment:
is this todo for this PR or later?

Reply:
I wasn't planning on making that in this pr and I doubt anyone else will, so worst case scenario this is a fix for only 1.19+ and the version check can be removed when we drop support for 1.18.

Reply:
Im fine with that. We then have solved the issue for over 75% of the servers atleast